minimizer/minimize.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+FILE="$1";
+CONDITION_SCRIPT="$2";
+shift;
+shift;
+CONDITION="$*";
+
+SCRIPT_PATH=$(cd "$(dirname "${BASH_SOURCE[0]}")" || exit; pwd -P);
+
+if ! [[ -x $(command -v dustmite) ]]; then
+    echo "No DustMite installed. It is typically contained in a package called 'dtools'.";
+    exit 1;
+fi
+
+if ! test -f "$FILE"; then
+    echo "Input file '$FILE' not found.";
+    exit 1;
+fi
+
+WORKSPACE=$SCRIPT_PATH/workspace
+mkdir "$WORKSPACE";
+cp "$FILE" "$WORKSPACE"/input.c;
```

Comment:
Maybe make this a symbolic link to reduce IO?
```suggestion
ln -s "$FILE" "$WORKSPACE"/input.c;
```